src/game_objects.py

```diff
@@ -30,7 +33,16 @@ def __init__(self, object: ACMIObject, color: pygame.Color = pygame.Color(255,25
             font = pygame.font.SysFont("couriernewbold", 22)
         self.font = font
         
-        self.color = pygame.Color(self.data.Color) 
+        self.color = pygame.Color(self.data.Color)
+
+        # Switch the object's color from its default to a respective replacement color
+        for i in config.app_config.get("map", "unit_color_switching", list):
+            print(self.color, "==", i[0])
+            if self.color == pygame.Color(i[0]):
+                if type(i[1]) != tomlkit.items.Array:
```

Comment:
Typically, explicit checking for type equality is not the best way to solve whatever problem it's trying to solve.

At the very least, you could use `isinstance()`, that way subclasses will also evaluate to true.

I'm not familiar with tomlkit though. Is this how it's typically done or something?

Reply:
No, it's dumber than that, I simply didn't know about `isinstance()` 🙃

Reply:
No problem, that's the benefit of code review, we learn things from our peers!